fastlane/Fastfile

```diff
@@ -99,13 +117,45 @@ ENV["PUBLIC_CONFIG_FILE"]="./config/Version.Public.xcconfig"
     ios_final_tag(options)
   end
 
-platform :ios do
-  desc "Push a new beta build to TestFlight"
-  lane :beta do
-    build_app(workspace: "Simplenote.xcworkspace", scheme: "Simplenote")
-    upload_to_testflight
+  #####################################################################################
+  # build_and_upload_beta
+  # -----------------------------------------------------------------------------------
+  # This lane builds the app and upload it for beta testing
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_upload_beta [skip_confirm:<skip confirm>]
+  #
+  # Example:
+  # bundle exec fastlane build_and_upload_beta 
+  # bundle exec fastlane build_and_upload_beta skip_confirm:true 
+  #####################################################################################
+  desc "Builds and updates for beta distribution"
+  lane :build_and_upload_beta do | options |
+    build_app(
```

Comment:
I hit an error here because my pods weren't up to date – do we want to run `pod install` as part of this?

Reply:
Yep! You're right. I added that and some other checks.